docs/src/main/asciidoc/dev-ui.adoc

```diff
@@ -887,6 +887,19 @@ JsonRPCProvidersBuildItem createJsonRPCServiceForCache() {// <2>
 
 https://github.com/quarkusio/quarkus/blob/main/extensions/cache/deployment/src/main/java/io/quarkus/cache/deployment/devui/CacheDevUiProcessor.java[Example code]
 
+Also cleanup this in Prod mode, so when the app build in prod mode, we remove all dev-ui related runtime classes:
```

Comment:
this pops up kinda in the middle in jsonrpc explanation. Maybe have a more explicit section/title like "Remove devUI from Production" ?

Reply:
It only really apply to Dev UI that has jsonrpc services. Many extensions does not have any. Any screen (js) and any build time data or runtime against the deployment cp is already not included in the prod build, it's just when you do the jsonrpc service, hence its added here